### PR TITLE
70533 super admin organization

### PR DESCRIPTION
**Pull request by adler99**

### CODE REVIEW

File: app/controllers/kadmin/application_controller.rb [hunk 1]
```diff
@@ -57,30 +57,18 @@ def handle_error(error, options = {})
     #
     # organization_scoped_ar is an ActiveRecord that has organization_scope(Organization) scope defined
     def scoped_find_by!(organization_scoped_ar, id)
-      if authorized_user.admin?
-        if id.is_a?(Array)
-          return organization_scoped_ar.find(id)
-        else
-          return organization_scoped_ar.find_by!(id: id)
-        end
+      if id.is_a?(Array)
```
Comment (sashamussich) [inline app/controllers/kadmin/application_controller.rb]: You dont really need check if id is []. This is enough:
```organization_scoped_ar.find(id)```

File: app/controllers/kadmin/auth_controller.rb [hunk 2]
```diff
@@ -60,6 +60,14 @@ def unauthorized
       }
     end
 
+    # POST /change_organization
+    def change_organization
+      if authorized_user && authorized_user.admin?
```
Comment (sashamussich) [inline app/controllers/kadmin/auth_controller.rb]: ```if authorized_user&.admin?```

File: lib/kadmin/auth/user.rb [hunk 3]
```diff
@@ -5,16 +5,21 @@ class User
...
+        @organization = organization if self.admin?
+      end
+
       def authorized?(_request)
         return true
       end
 
       def admin?
-        return true
+        return @admin
```
Comment (sashamussich) [inline lib/kadmin/auth/user.rb:22]: What is returned here? Boolean value?

**Review by sashamussich — CHANGES_REQUESTED** [review]

File: lib/kadmin/auth/user.rb (hunk 3 above)
Comment (adler99) [reply lib/kadmin/auth/user.rb:22]: yes :-)

File: app/controllers/kadmin/application_controller.rb (hunk 1 above)
Comment (adler99) [reply app/controllers/kadmin/application_controller.rb]: done, thanks for the review!

File: app/controllers/kadmin/auth_controller.rb (hunk 2 above)
Comment (adler99) [reply app/controllers/kadmin/auth_controller.rb]: done

**Review by sashamussich — APPROVED** [review]

**Pull request merged**